### ISSUE TITLE
DataLoader, the first 'y' different from others

### ISSUE THREAD

**Issue by MSDP4DOM** [chrsmrrs/k-gnn#21]
Dear authors, 

This is my first post of a github issue, please forgive me for the improper format. I'm tring to reproduce the work 'Graph_Neural_Network_for_Fuel_Ignition_Quality' (https://git.rwth-aachen.de/avt-svt/public/graph_neural_network_for_fuel_ignition_quality).  I replaced the original database in the repository with a boiling point database. Then the problem comes, after the database was created using:
`dataset = PropDataset(
        path + 'Train/', 
        pre_transform=MyPreTransform(),
        pre_filter=MyFilter())；
dataset.data.iso_type_2 = torch.unique(dataset.data.iso_type_2, True, True)[1]；
num_i_2 = int(dataset.data.iso_type_2.max().item() + 1)；
dataset.data.iso_type_2 = F.one_hot(dataset.data.iso_type_2, num_classes=num_i_2).to(torch.float)`

At this stage, data was loaded correctly
`dataset.data.y[:5]
Out[87]: 
tensor([[413.8000],
        [413.1500],
        [574.1500],
        [476.1500],
        [608.1500]])`

A further step is applied to normilize the dataset: 
`mean = torch.as_tensor(dataset.data.y, dtype=torch.float).mean()；
std = torch.as_tensor(dataset.data.y, dtype=torch.float).std()；
dataset.data.y = (dataset.data.y - mean) / std`

At this stage, the original data is proved to be normalized, as indicated by the return of input `dataset.data.y[:5]
Out[66]: 
tensor([[-0.5562],
        [-0.5634],
        [ 1.2268],
        [ 0.1371],
        [ 1.6049]])`

however, when loading the 'database', problem comes out, all normalized y are loaded correctly except the first one _413.80_, which is exactly the same with original database before normalization applied, but different to the normalized one -0.5562:
`from k_gnn import DataLoader;
all_loader = DataLoader(dataset, batch_size = 64)；
for i, dataset1 in enumerate(all_loader):
    if i <= 1:
        print(dataset1.y[:5])
tensor([[ 4.1380e+02],  
        [-5.6341e-01],
        [ 1.2268e+00],
        [ 1.3712e-01],
        [ 1.6049e+00]])
tensor([[-1.2050],
        [-1.4974],
        [-1.4407],
        [-0.0186],
        [ 0.5263]])`

Could you please help me find out the reasons? many thanks!

Best regards,
Patrick

**Comment by MSDP4DOM** [comment]
Follow up, after restart the computer and Pycharm IDE, the results is back to normal. 
Inputs: 

`dataset = PropDataset(
    path + 'Train/',
    pre_transform=MyPreTransform(),
    pre_filter=MyFilter())
dataset.data.iso_type_2 = torch.unique(dataset.data.iso_type_2, True, True)[1]
num_i_2 = int(dataset.data.iso_type_2.max().item() + 1)
dataset.data.iso_type_2 = F.one_hot(dataset.data.iso_type_2, num_classes=num_i_2).to(torch.float)

print('----1:\n',dataset.data.y[:5])

mean = torch.as_tensor(dataset.data.y, dtype=torch.float).mean()
std = torch.as_tensor(dataset.data.y, dtype=torch.float).std()
dataset.data.y = (dataset.data.y - mean) / std

print('----2:\n',dataset.data.y[:5])

from k_gnn import DataLoader
print('----3:\n')
all_loader = DataLoader(dataset, batch_size = 64)
    for i, dataset1 in enumerate(all_loader):
        if i <= 1:
            print(dataset1.y[:5])`


Returned results:

`----1:
 tensor([[413.8000],
        [413.1500],
        [574.1500],
        [476.1500],
        [608.1500]])
----2:
 tensor([[-0.5562],
        [-0.5634],
        [ 1.2268],
        [ 0.1371],
        [ 1.6049]])
----3:
tensor([[-0.5562],
        [-0.5634],
        [ 1.2268],
        [ 0.1371],
        [ 1.6049]])
tensor([[-1.2050],
        [-1.4974],
        [-1.4407],
        [-0.0186],
        [ 0.5263]])`
